erna/__init__.py

```diff
@@ -18,8 +18,32 @@ def build_path(row, path_to_data, extension):
     year = night[0:4]
     month = night[4:6]
     day = night[6:8]
-    return os.path.join(path_to_data, year, month, day, row.filename + extension)
-
+    res = os.path.join(path_to_data, year, month, day, row.filename + extension)
+    return res
+
+def test_drs_path(df, key):
```

Comment:
This one is called `test_drs_path`, the function below is called `test_path_data`. This is inconsistent.

Reply:
Missing docs